### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by mathiasbno**

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -29,6 +30,10 @@ blockquote {
   padding: 0;
   transform: translate(-50%, -50%);
   text-align: center;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```
Comment (mathiasbno) [inline public/styles.css:49]: We might not want this, but found that it selected text when clicking through the site.

Comment (sindrenm) [inline public/styles.css:49]: I would probably let people select text.

**Comment by jgorset** [comment]
You get :cake:!

**Comment by sindrenm** [comment]
What‽ Seriously, I would really like to have the possibility to select text. (https://github.com/hyperoslo/hyper-says/pull/1#discussion-diff-42916725)

**Comment by jgorset** [comment]
WELL THATS TOO BAD

**Comment by jgorset** [comment]
Seriously though, I don't feel strongly about it – I just know that I haven't had the need to select text, and that I have been annoyed by my browser thinking I wanted to do that.

**Comment by jgorset** [comment]
It'd be nice if this pull request was actually three pull requests so we could discuss that change and still be able to merge the others, though, @mathiasbno. Feel free to open a pull request to revert that particular change if you'd really like to have the opportunity to select text, @sindrenm. :wink:

**Pull request merged**